cypress/fixtures/search-result/facet-browser/intelligentFacets.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Please move this file so it is co-located with there it is used.

This file is placed in the /cypress/fixtures/facet-browser/ directory but it is used by code in the `facet-line` tests.